### DISCUSSION TITLE
Find and replace paths

### DISCUSSION THREAD

**Discussion by AtomicNess123** [BurntSushi/ripgrep#2429] [General] [answered]
How to modify 

`rg foo --files-with-matches | xargs sed -i '' 's/foo/bar/g'`

when `foo` and `bar` are paths which contain `/` ?

Thanks!

**Comment by BurntSushi** [comment, marked as answer]
That's a sed question, not a ripgrep question. But you can either escape the slash with `\/`, or use different delimiters for your sed expression, e.g., `s#foo#bar#g`.
